src/test/java/no/nav/veilarbvedtaksstotte/utils/AbstractVedtakIntegrationTest.kt

```diff
@@ -49,7 +49,8 @@ abstract class AbstractVedtakIntegrationTest {
         gjeldende: Boolean = true,
         enhetId: String = "1234",
         veilederIdent: String = "VIDENT",
-        beslutterIdent: String? = null
+        beslutterIdent: String? = null,
+        gammelVedtakId : Long = 1234
```

Comment:
Kanskje man kunne hentet ut det gamle/gjeldende vedtaket i denne funksjonen i stede for å sende inn `gammelVedtakId`?